### DISCUSSION TITLE
<Link /> wrapping <a /> without href in <a /> triggers `eslintjsx-a11y/anchor-is-valid`

### DISCUSSION THREAD

**Discussion by patarapolw** [vercel/next.js#13441] [Help] [answered]
The syntax is

```tsx
<Link href="/"><a><h1>{titleRoot}</h1></a></Link>
```

![Screen Shot 2563-05-27 at 20 28 29](https://user-images.githubusercontent.com/21255931/83024571-cae82880-a058-11ea-8387-ac836b21a9dc.png)

**Comment by Timer** [comment, marked as answer]
Please see #8207
